bin/install.sh

```diff
@@ -89,34 +101,44 @@ env OKTA_PROFILE=$profile java \
     -Djava.util.logging.config.file=~/.okta/logging.properties \
     -classpath ~/.okta/okta-aws-cli.jar \
     com.okta.tools.WithOkta $command $@
-' > "$PREFIX/bin/withokta"
-chmod +x "$PREFIX/bin/withokta"
+EOF
+chmod +x "${dotokta}/bin/withokta"
 
 # Create okta-credential_process command
-echo '#!/bin/bash
+cat <<'EOF' >"${dotokta}/bin/okta-credential_process"
+#!/bin/bash
 roleARN="$1"
 shift;
 env OKTA_AWS_ROLE_TO_ASSUME="$roleARN" \
     java -classpath ~/.okta/okta-aws-cli.jar com.okta.tools.CredentialProcess
-' > "$PREFIX/bin/okta-credential_process"
-chmod +x "$PREFIX/bin/okta-credential_process"
+EOF
+chmod +x "${dotokta}/bin/okta-credential_process"
 
 # Create okta-listroles command
-echo '#!/bin/bash
+cat <<EOF >"${dotokta}/bin/okta-listroles"
+#!/bin/bash
 java -classpath ~/.okta/okta-aws-cli.jar com.okta.tools.ListRoles
-' > "$PREFIX/bin/okta-listroles"
-chmod +x "$PREFIX/bin/okta-listroles"
+EOF
+chmod +x "${dotokta}/bin/okta-listroles"
 
 # Configure Okta AWS CLI
-oktaConfig="${HOME}/.okta/config.properties"
-grep '^#OktaAWSCLI' "${oktaConfig}" > /dev/null 2>&1
-if [ $? -ne 0 ]
-then
-echo "
+oktaConfig="${dotokta}/config.properties"
```

Comment:
This won't work unless PREFIX is ~/.okta/ which complicates system installs.

The location of config.properties is effectively hard-coded here:
https://github.com/oktadeveloper/okta-aws-cli-assume-role/blob/master/src/main/java/com/okta/tools/OktaAwsConfig.java#L96-L108

suggestion:
```suggestion
oktaConfig="${HOME}/.okta/config.properties"
```